<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingCategoryDialog</class>
 <widget class="QDialog" name="SettingCategoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>CategoryID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPhase">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPriority">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Result name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditResultName">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>L0</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinLengthL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>40</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxLengthL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxPixelsL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>100</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min pixels</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinPixelsL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffHL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>160</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff high</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>130</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff Low</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffLL0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>L1</string>
    </attribute>
    <widget class="QSpinBox" name="spinBoxMinPixelsL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffLL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxPixelsL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>100</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>130</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff Low</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffHL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxLengthL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>40</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max length</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>160</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff high</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinLengthL1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>L2</string>
    </attribute>
    <widget class="QSpinBox" name="spinBoxMinPixelsL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffLL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxPixelsL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>100</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>130</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff Low</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBrightDiffHL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxLengthL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>40</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Max length</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>160</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Brightness diff high</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Min length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinLengthL2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonAddList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Category ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCategoryID">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
